.test-infra/tools/flaky_test_detection.py

```diff
@@ -24,6 +24,18 @@
 ALERT_NAME = "flaky_test"
 GIT_ORG = "apache"
 GRAFANA_URL = "http://metrics.beam.apache.org"
+GRAFANA_PANEL_IDS = {
+    "core_infra": 1,
+    "core_java": 2,
+    "dataflow_java": 3,
+    "runners_java": 4,
+    "load_perf_java": 5,
+    "core_python": 6,
+    "runners_python": 7,
+    "load_perf_python": 8,
+    "go": 9,
+    "misc": 10,
```

Comment:
Could we add important? I added this in https://github.com/apache/beam/pull/30920 - though I seemingly messed something up as part of that since it hasn't populated yet - http://metrics.beam.apache.org/d/CTYdoxP4z/ga-post-commits-status?orgId=1

If you see what's wrong with the current setup that would be helpful too! I think I might've duplicated ids?

Reply:
Added "important_signals" category and reordered dashboard ids. Also, I deployed the GCP [function](https://console.cloud.google.com/functions/details/us-central1/github_workflow_prefetcher?env=gen1&project=apache-beam-testing&tab=source) so your changes are now applied

Reply:
Ah thanks - forgot to do that!